README.md

```diff
@@ -26,6 +26,34 @@ Returns output that looks like:
 
     Kubernetes master is running at https://146.148.80.79
 
+In our configuartion, we also rely on persistent disks created by the Google Cloud Platform for storage. You may provision a disk on the Google Cloud Shell using the following command:
```

Comment:
nit: `configuration`

nit: Can you also limit the length of your lines to 80 characters? Just for consistency sake. You can see where the limit is with:

```
"rulers":
[
	79
],
```
In your sublime config.